opal/resource.go

```diff
@@ -180,6 +187,103 @@ func resourceResource() *schema.Resource {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     true,
+				Deprecated:  "Use request_configurations_list instead.",
+			},
+			"request_configuration": {
```

Comment:
if it's a list, do we want to pluralize?

Reply:
i think reviewer_stage is a list and singular (cuz it's used as a block so it should be singular)